refellips/tests/WVASE_example_30nmSiO2_90nmPNIPAM_20EMA-MG_MultiWavelength.txt

```diff
@@ -0,0 +1,42 @@
+
+400	60.00	34.444	164.09
```

Comment:
Can you shorten the file name, getting a bit long?
--> WVASE_30nmSiO2_90nmPNIPAM_20EMA-MG.txt

Reply:
Yes I can, but I tried to keep the file names as descriptors for the system they represent. This is consistent across all demo/test files. Do we want to shorten all file names?

Reply:
It just gets a bit harder for me to figure out which file is which. It's not a killer tho.